languages/JavaScript/exercises/concept/arrays-basic/.docs/introduction.md

```diff
@@ -0,0 +1,8 @@
+In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or ojects, even mixed types. The array elements can be accesed by their index.
+
+Examples of equivalent declaration:
+
+```javascript
+const numbers = [1, 'two', 3, 'four']; // most common`
+const numbers = new Array(1, 'two', 3, 'four');
```

Comment:
Maybe this code example (or a a new one) can show how to access an element by index, as that has a specific syntax and we want to have examples for all newly introduced syntax.